City-of-Helsinki/employment-services-ui
# THF-521: Add null check to images

Added null check to the event images

how to test:

- Go to event page and go bottom of the event list "load more" so long that you get an error.
- Go to page (Event without image) http://localhost:3000/en/current-matters/events/nyt-tarjolla-monenlaisia-toita-rakennusalalta
- check also front page lift up that it still works
